.env.io-onboarding-pa-api

```diff
@@ -0,0 +1,9 @@
+CLIENT_SPID_ERROR_REDIRECTION_URL=/error.html
```

Comment:
this file is not needed, env vars will be set in the helm chart

Reply:
if these variables are not set the container doesn't even start. These are simply demo values to use for local development

Reply:
then the file should have the `.development` suffix